src/Altinn.Platform/Altinn.Platform.Events/Tests/TestingControllers/EventsControllerTests.cs

```diff
@@ -379,37 +416,6 @@ public async void Get_AfterIncludedInQuery_ReturnsNextHeaderWithReplacesAfterPar
                 Assert.Equal(expectedNext, response.Headers.GetValues("next").First());
             }
 
-            /// <summary>
```

Comment:
why was the test deleted?

Reply:
Because it is not a valid testscenario since a user can call this

Reply:
I would like to see a test that verifies the logic that register is called if orgnumber or ssn is included in the query. I don't think that is covered when we remove this test. How about using an org number instead org an ssn if that is more of a valid test case?